static/crisp.js

```diff
@@ -0,0 +1,13 @@
+window.$crisp = [];
```

Comment:
What is this doing? Resetting crisp?

Reply:
Yeah, they use it to set active sessions etc. It's part of their official snippet.